docs/CHANGELOG.rst

```diff
@@ -4,6 +4,19 @@ Change Log
 
 All notable changes to this project are documented in this file.
 
+===========
+Unreleased:
+===========
+
+Changed
```

Comment:
I would prefer if you deleted unused sections (i.e. *Changed* and *Fixed*) and add them if/when they are necessary. Similar to what I've done in genialis/resolwe#49.